app/assets/javascripts/sephora_style_guide/video.js

```diff
@@ -29,35 +18,23 @@ function onYouTubeIframeAPIReady() {
 
 function onPlayerReady() {
   // bind events
-  $(document).on('click', '.close', function() {
-    var currentVideoId = $(this).closest('.modal-header').siblings().find('iframe').attr('id');
+  $(document).on('click', '.video-link', function() {
+    var currentVideoId = $(this).attr('data-video');
 
     for (var i in players) {
       if (players[i].a.id === currentVideoId) {
-        players[i].pauseVideo();
+        players[i].playVideo();
       }
     }
   });
 
-  $(document).on('click', '.modal[role="dialog"]', function() {
+  $('.modal[role="dialog"]').on('hide.bs.modal', function() {
```

Comment:
Don't video modals have any specific class that can be used for this? Just thinking that this will now apply to every single modal on the site. Maybe a suggestion is to use `js-video-modal`, then add that to the code snippet too.